### PR TITLE
Editorial: Add single line character count "word" exception

### PR DESCRIPTION
**Pull request by snuggs**
Addresses: https://github.com/whatwg/meta/issues/37#issuecomment-342587779

### CODE REVIEW

File: COMMITTING.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ For more on this topic, see ["On commit messages"](http://who-t.blogspot.com/200
 
 Commit messages consist of a single title line, a blank line, and then a more detailed change description. The description (and preceding blank line) may be omitted for simple fixes.
 
-All lines should be at most 72 characters, to make them easier to view in GitHub and other tools. Some sources recommend restricting the title line to 50 characters, but we do not enforce this.
+All lines should be at most 72 characters, to make them easier to view in GitHub and other tools. Single "words" are an exception to this convention (i.e. URLs). Some sources recommend restricting the title line to 50 characters, but we do not enforce this.
```
Comment (annevk) [inline COMMITTING.md]: I think it should be "e.g., URLs".

**Comment by snuggs** [comment]
@annevk please do a brief scan as i found a couple instances that needed updating. Besides this should be good to go. Thanks for the nit! 💪  Enjoy your day sir.

File: COMMITTING.md [hunk 2]
```diff
@@ -25,15 +25,15 @@ For more on this topic, see ["On commit messages"](http://who-t.blogspot.com/200
...
 
-The title must be written in imperative mode, as if commanding someone. This means using verb conjugations such as "fix"/"add"/"change" instead of "fixed"/"added"/"changed" or "fixing"/"adding"/"changing". We are less consistent with the description, but often use present tense (i.e. "This change adds…").
+The title must be written in imperative mode, as if commanding someone. This means using verb conjugations such as "fix"/"add"/"change" instead of "fixed"/"added"/"changed" or "fixing"/"adding"/"changing". We are less consistent with the description, but often use present tense (e.g. "This change adds…").
 
 Title lines must not end in a period; they are titles, not sentences.
 
 Be sure to reference related GitHub issues within pull requests so they are appropriately cross-linked by GitHub. Use "closes" or "fixes" as appropriate to [automatically close issues](https://help.github.com/articles/closing-issues-using-keywords/).
 
-Avoid using Markdown-style code markup (i.e. backticks) unless necessary for disambiguation.
+Avoid using Markdown-style code markup (e.g. backticks) unless necessary for disambiguation.
```
Comment (annevk) [inline COMMITTING.md]: i.e. seems appropriate here?

**Comment by annevk** [comment]
They also need to be followed with a comma.

**Comment by annevk** [comment]
(Sorry for the terse replies, really appreciate this.)

**Comment by snuggs** [comment]
No worries at all @annevk. I have tough skin! #OSS101. I JUST read all "e.g., " must be followed by comma even if just one example". #TheMoreYouKnow ⭐️ 

Seems e.g., (comma) is **ALWAYS** and i.e., is **¯\\_(ツ)_/¯**
https://www.dailywritingtips.com/comma-after-i-e-and-e-g/

Fixed in ae6058

**Comment by annevk** [comment]
\o/

**Pull request merged**